### PR TITLE
updated drops downs, handled edge cases

### PR DESCRIPTION
**Pull request by mertbagt**
changed US State and Cong. Office drop-downs to match new Status Drop-Downs

handled a number of edge cases:

- US State to Cong. Office mismatching
- top option not updating onClick
- input control warning when adding a new order

added some messaging to remind users if their edits are currently unsaved

pointer to overall project info added to the readme

abbr. Mail Services to MAIL in Statuses

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...ze":6114,"mtime":1634836767243,"results":"10","hashOfConfig":"7"},{"size":1580,"mtime":1634836948412,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"gppgzb",{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"18","messages":"19","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"20","messages":"21","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\App.js",[],"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\AddOrder.js",[],"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\EditOrder.js",[],"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\OrderForm.js",[],"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\Statuses.js",[]]
```
Comment (ethanstrominger) [inline .eslintcache:1]: Can you add this file to the .gitignore and then run git rm <file>

File: src/components/AddOrder.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ const AddOrder = (props) => {
     home_office_code: "",
     usa_state: "",
     published: false,
+    // selection: "select", // to be uncommented when integrated into backend response handling, should be set consistently in AddOrder.js and Order.js
```
Comment (ethanstrominger) [inline src/components/AddOrder.js:14]: Can you remove this comment?

File: src/components/AddOrder.js [hunk 3]
```diff
@@ -19,6 +20,14 @@ const AddOrder = (props) => {
   );
   const [submitted, setSubmitted] = useState(false);
 
+  // responses from DB overwriting order.status_description, order.selection
```
Comment (ethanstrominger) [inline src/components/AddOrder.js:23]: Can you remove this comment?

File: src/components/OrderForm.js [hunk 4]
```diff
@@ -9,17 +9,21 @@ const OrderForm = (props) => {
...
     const { name, value } = event.target;
-    setOrder({ ...order, [name]: value });
-    if (resetMessage) {
-      resetMessage("");
+    if (mode !== "edit" && name === "usa_state") {
+      setOrder({ ...order, [name]: value, home_office_code: "" });
+    } else {
+      setOrder({ ...order, [name]: value });
+    }
+    if (updateMessage) {
```
Comment (ethanstrominger) [inline src/components/OrderForm.js:25]: Can we get on the phone to discuss this one?

File: src/components/OrderForm.js [hunk 5]
```diff
@@ -28,9 +32,23 @@ const OrderForm = (props) => {
...
+    updateMessage("");
   };
 
+  let districtMatchCheck = true; // putting this in Component State makes this check old state instead of what state is being updated to
+  if (mode === "edit") {
+    // and/or exceed maximum update depth error
+    let currentDistricts = STATES.filter(
+      (state) => state.name === order.usa_state
+    );
+    console.log("Current US State's Disticts: ", currentDistricts[0].districts);
```
Comment (ethanstrominger) [inline src/components/OrderForm.js:44]: Remove console.log

File: src/components/OrderForm.js [hunk 6]
```diff
@@ -47,13 +65,19 @@ const OrderForm = (props) => {
...
+          <strong>{order.usa_state ? order.usa_state : "**"}</strong>
+        </div>
         <select
-          value={order.usa_state}
+          value={status.selection} // change to {order.selection} after user db integrated
           id="usa_state"
           onChange={handleInputChange}
           name="usa_state"
         >
+          <option value="select" key="blank" hidden disabled>
```
Comment (ethanstrominger) [inline src/components/OrderForm.js:78]: When we meet, can you explain this tome?

File: .eslintcache (hunk 1 above)
Comment (mertbagt) [reply .eslintcache]: done

File: src/components/AddOrder.js (hunk 3 above)
Comment (mertbagt) [reply src/components/AddOrder.js:23]: done

**Pull request merged**